### PR TITLE
Cloud multi-user docs

### PR DESCRIPTION
**Pull request by pierwill**
Documents inviting, removing users. Gives information on permissions (currently Owners only).

### CODE REVIEW

File: content/v2.0/account-management/multi-user/_index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+description: >
+  View and manage multiple users in an Influx Cloud account.
+weight: 4
+menu:
+  v2_0:
+    parent: Account management
+    name: Manage multiple users
+---
+
+{{< cloud-name >}} accounts support multiple-user organizations.
```
Comment (kelseiv) [inline content/v2.0/account-management/multi-user/_index.md:12]: multiple users in an organization

File: content/v2.0/account-management/multi-user/_index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Collaborate with others using these features.
```
Comment (kelseiv) [inline content/v2.0/account-management/multi-user/_index.md:13]: By default, each user has full permissions on resources in your organization.

File: content/v2.0/account-management/multi-user/invite-user.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+description: >
+  Invite a user to collaborate
+weight: 103
+menu:
+  v2_0:
+    parent: Manage multiple users
+    name: Invite a user
+---
+
+## Invite user from {{< cloud-name >}}
```
Comment (kelseiv) [inline content/v2.0/account-management/multi-user/invite-user.md:13]: drop line 13 to eliminate the double heading for now...when we have  multiple Cloud/OSS sections, we can add intro sentence and new headers

File: content/v2.0/account-management/multi-user/invite-user.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+1. Navigate to the **Members** page under **Organizations** in the left navigation bar.
+2. Under "Add a new member to your organization," enter the email address of the user to invite.
```
Comment (kelseiv) [inline content/v2.0/account-management/multi-user/invite-user.md:16]: Under **Add a new member...**

File: content/v2.0/account-management/multi-user/invite-user.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+   (Users must be invited one at a time.)
+3. Click **Add & Invite**.
+
+An invitation with an activation link will be sent to the specified email address.
```
Comment (kelseiv) [inline content/v2.0/account-management/multi-user/invite-user.md:20]: is sent

File: content/v2.0/account-management/multi-user/invite-user.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+Once activated, the new user will be added as an **Owner** with permissions to read and write all resources.
+
+{{% warn %}}
+Take care when inviting a user as an **Owner**.
+Owners can delete resources and other users.
+{{% /warn %}}
+
+{{% cloud %}}
+Currently {{<cloud-name >}} has only one permission level.
+Future releases will contain read-only users.
```
Comment (kelseiv) [inline content/v2.0/account-management/multi-user/invite-user.md:31]: and role based permissions?

Comment (kelseiv) [inline content/v2.0/account-management/multi-user/invite-user.md:31]: I might combine lines 24-31 and move to top of page. Perhaps:

{{% warn %}}
Currently, Cloud 2.0 has only one permission level: **Owner**.
With Owner permissions, a user can delete resources and other users from your organization. Take care when inviting a user. 
Future releases will contain multiple permission levels.
{{% /warn %}}

File: content/v2.0/account-management/multi-user/remove-user.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+description: >
+  Remove a user from an organization
+weight: 103
+menu:
+  v2_0:
+    parent: Manage multiple users
+    name: Remove a user
+---
+
+## Remove a user from {{< cloud-name >}}
```
Comment (kelseiv) [inline content/v2.0/account-management/multi-user/remove-user.md:13]: Curious why not use "delete" ?

File: content/v2.0/account-management/multi-user/invite-user.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+weight: 103
+menu:
+  v2_0:
+    parent: Manage multiple users
+    name: Invite a user
+---
+
+## Invite user from {{< cloud-name >}}
+
+1. Navigate to the **Members** page under **Organizations** in the left navigation bar.
```
Comment (sanderson) [inline content/v2.0/account-management/multi-user/invite-user.md:15]:
```
1. In the navigation bar on the left, select **Organizations** > **Members**.

    {{< nav-icon "org >}}
```

File: content/v2.0/account-management/multi-user/invite-user.md (hunk 5 above)
Comment (sanderson) [inline content/v2.0/account-management/multi-user/invite-user.md:20]: ~be sent~ -> is sent

File: content/v2.0/account-management/multi-user/invite-user.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+2. Under "Add a new member to your organization," enter the email address of the user to invite.
+   (Users must be invited one at a time.)
+3. Click **Add & Invite**.
+
+An invitation with an activation link will be sent to the specified email address.
+
+Once activated, the new user will be added as an **Owner** with permissions to read and write all resources.
```
Comment (sanderson) [inline content/v2.0/account-management/multi-user/invite-user.md:22]: ~will be~ -> is

File: content/v2.0/account-management/multi-user/invite-user.md (hunk 6 above)
Comment (sanderson) [reply content/v2.0/account-management/multi-user/invite-user.md:31]: I hesitate to document _future_ functionality. This has always come back to burn me. I think we just say, "Currently {{< cloud-name >}} only supports read-write permissions for members."

File: content/v2.0/account-management/multi-user/remove-user.md (hunk 7 above)
Comment (sanderson) [reply content/v2.0/account-management/multi-user/remove-user.md:13]: "Delete" implies that the user is completely removed from the system. They aren't. They just don't have access to your organization, but they may belong to other organizations.

File: content/v2.0/account-management/multi-user/remove-user.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
+---
+title: Remove a user
+list_title: Remove a user
+description: >
+  Remove a user from an organization
```
Comment (sanderson) [inline content/v2.0/account-management/multi-user/remove-user.md:5]: Remove a (user/member?) from an **InfluxDB Cloud** organization.

File: content/v2.0/account-management/multi-user/remove-user.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+weight: 103
+menu:
+  v2_0:
+    parent: Manage multiple users
+    name: Remove a user
+---
+
+## Remove a user from {{< cloud-name >}}
+
+1. Navigate to the **Members** page under **Organizations** in the left navigation bar.
```
Comment (sanderson) [inline content/v2.0/account-management/multi-user/remove-user.md:15]:
```
1. In the navigation bar on the left, select **Organizations** > **Members**.

    {{< nav-icon "org >}}
```

**Comment by pierwill** [comment]
Thanks, Scott! I'll review the member/user terms with the team—

File: content/v2.0/account-management/multi-user/remove-user.md (hunk 7 above)
Comment (kelseiv) [reply content/v2.0/account-management/multi-user/remove-user.md]: Does the UI say delete? Perhaps rename procedure "Delete user from organization"?
In context of an organization, delete and remove imply the same. How many users are going to be part of multiple orgs?

**Pull request merged**